packages/discussions/graphql/resolvers/Discussion/comments.js

```diff
@@ -245,6 +244,29 @@ module.exports = async (discussion, args, context, info) => {
     }
   }
 
+  /* 
+    AUTO = comments are sorted 
+    - by date if the first comment was created in the last 24 hours 
+    - or by votes otherwise 
+    the property resolvedOrderBy is just needed when DiscussionOrder === AUTO
+  */
+  let resolvedOrderBy
```

Comment:
```suggestion
  let resolvedOrderBy = 'DATE'
```

Maybe default to `DATE` here and remove else-clause further below.

Reply:
kept it undefined, because when `orderBy !== 'AUTO'` it should not return a value